types/i2c-bus/index.d.ts

```diff
@@ -33,8 +44,7 @@ export interface I2cBusFuncs {
     smbusWriteI2cBlock: boolean;
 }
 
-export interface I2cBus {
-
+export interface Bus {
```

Comment:
I think it would be nice to keep the old name for those updating the typings to the latest version.  Also, if people are using other hardware, there might be a collision of type names if it is left so general.

Reply:
Thanks for reviewing this PR. Fixed in c0e7903.